akka-actor-testkit-typed/src/test/scala/akka/actor/testkit/typed/scaladsl/BehaviorTestKitSpec.scala

```diff
@@ -456,7 +458,7 @@ class BehaviorTestKitSpec extends AnyWordSpec with Matchers with LogCapturing {
       testkit.expectEffectPF {
         case Effect.Spawned(_, "child", _) =>
       }
-      //no effect since the timer's mode was single, hence removed after fired
+      // no effect since the timer's mode was single, hence removed after fired
```

Comment:
maybe it's ok to drop the ` `.

Reply:
So this appears to be impossible with newer version of scalafmt. It always puts a space after the `//` and there is no configuration to disable this (I believe this is deliberate because at least from what I have seen pretty much everyone adds spaces after `//`, at least in Scala).